docs/software/system_stats.md

```diff
@@ -417,4 +418,51 @@ Official documentation: <https://rancher.com/docs/k3s/latest/en/>
 Source code: <https://github.com/k3s-io/k3s>  
 License: [Apache 2.0](https://github.com/k3s-io/k3s/blob/master/LICENSE)
 
+## Microk8s
+
+High availability - Low-ops, minimal production Kubernetes, for devs, cloud, clusters, workstations, Edge and IoT.
+
+![microk8s logo](../assets/images/microk8s.png){: width="300" height="116" loading="lazy"}
+
+
+
+=== "Connecting to your cluster"
+
+    To create a cluster out of two or more already-running MicroK8s instances, use the microk8s add-node command
+
+    ```sh
+    microk8s add-node
+    From the node you wish to join to this cluster, run the following:
+    microk8s join 192.168.1.230:25000/92b2db237428470dc4fcfc4ebbd9dc81/2c0cb3284b05
+    Use the '--worker' flag to join a node as a worker not running the control plane, eg:
+    microk8s join 192.168.1.230:25000/92b2db237428470dc4fcfc4ebbd9dc81/2c0cb3284b05 --worker
+    ```
+
+    for most commands, you can use microk8s in front like below.
+
+    ```sh
+    microk8s kubectl get nodes
+    microk8s kubectl get napspaces
+    ```
+
+    Microk8s does provide a few "addons", which can be seen below with the enable and disable command.
+
+    ```sh
+    microk8s status #to view the addons
+    microk8s enable dns # to enable addons
+    microk8s enable dashboard # to enable kubernetes dashboard
+    ```
+
+=== "View logs"
+
+    - Service: `journalctl -u microk8s`
```

Comment:
Since there is no `microk8s.service`, this is always empty. `journalctl` shows all system logs, including those of MicroK8s and Snap. Since there are a lot of complexly named services involved, it is very difficult to reasonably filter it. There are per-container log files in `/var/log/pods`, but quite split, difficult to get an overview of what happened, also without further readings, it is not clear which pod is part of MicroK8s core, where hence would be one to have a look at if startup fails or such.

@silverraindog 
Which logs did you have a look at for debugging MicroK8s in general, if it was necessary already? Otherwise I'd suggest `journalctl` (without filtering) and `/var/log/pods` for looking at individual component logs.